google-cloud-spanner-hibernate-dialect/pom.xml

```diff
@@ -18,5 +18,10 @@
       <groupId>org.hibernate</groupId>
       <artifactId>hibernate-core</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-spanner-jdbc</artifactId>
```

Comment:
How come we didn't need this before? I thought the idea was the that the user adds the jdbc dependency.

Reply:
Yeah I only needed it to use the `Code` enum provided by the spanner libs. I'm using this enum in the [ArrayJavaTypeDescriptor](https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/pull/232/files#diff-7ffc33f6760659087bc9c552f081c3d9a90a5fe2be9e4fffdf3eaf96697630ffR140) to define the mapping between Java types and Spanner SQL codes.

I could also just map the Java type to a hardcoded String; but just thought using the Code enum would be cleaner. Tho if it's not worth having the dependency then I can use String.

Also something to note: I think the Spanner JDBC driver is much more mature than Simba, so users won't be choosing between the two. Maybe it makes sense to just include the spanner JDBC driver now.

Reply:
We only support the official driver anyway. So, I think it's fine. We can still leave the instructions to add the dependency to the app so that users can control the version independently.

Reply:
Got it.